### DISCUSSION TITLE
Expose TriggerMessage

### DISCUSSION THREAD

**Discussion by rbb-dev** [lbbrhzn/ocpp#1445] [Ideas]
Hi,

Just wondering if it is possible to add TriggerMessage to this project.
I want to send triggers (MeterValues, etc) to changers shortly after changing profiles to update values faster than what's configured.

**Comment by drc38** [comment]
Yes it can be done, you can follow the other services examples in order to add it https://github.com/lbbrhzn/ocpp/blob/d351da0c7cc7f4a0dcc135e6ddffe2d9a5814b1b/custom_components/ocpp/chargepoint.py#L346
